event_survey/wizard/event_survey_wizard.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2019 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import fields, models, _
+
+
+class WizardEventSurvey(models.TransientModel):
+
+    _name = 'wizard.event.survey'
```

Comment:
My 2 cents, but IMO would make sense to name the wizard with what it does instead of just calling it 'Wizard', e.g `event.survey.generator` gives a better idea of what it does without having to check the code or try it out. (and it's also closer to the name of its action https://github.com/camptocamp/odoo-event-addons/pull/24/files#diff-30bfa33a8eb2f231d2a8a66f8f3b7621R25) :wink: 

Moreover, keeping `event.survey.*` as prefix of your model name, and adding the rest after keeps it in line with odoo naming convention and OCA guidelines.

Reply:
I agree w/ @grindtildeath.  If you want to keep something that mentions clearly this is a wizard you can append `wiz` to it, like `event.survey.generator.wiz`